arches/app/media/js/views/components/widgets/number.js

```diff
@@ -18,6 +18,15 @@ define(['knockout', 'underscore', 'viewmodels/widget', 'bindings/formattedNumber
 
         var self = this;
 
+        this.disable = ko.computed(() => {
+            if (ko.isObservable(self.disabled)) {
```

Comment:
This is functional, but I think that we should default to `uneditable` when `disabled` is unavailable, ie something like this:
```
let disable = self.uneditable();
if (ko.isObservable(self.disabled)) {
    disable = disable || self.disabled();
}
return disable;
```